### PR TITLE
[EAGLE-4085] Add Model Export

### PR DESCRIPTION
**Pull request by sainivedh**
### Why

* Support Model Export

### How

```python
 >>> from clarifai.client.model import Model
 >>> model = Model("url")
 >>> model.export('/path/to/export_model_dir')
```
### Tests

* Manual test
Screenshot 2024-03-13 at 12 29 49 AM

### Notes

*

### CODE REVIEW

**Review by ackizilkale — APPROVED** [review]
this looks good to me!

File: clarifai/client/model.py [hunk 1]
```diff
@@ -757,3 +758,90 @@ def get_latest_eval(self,
...
+
+    Args:
+        export_dir (str): The directory to save the exported model.
+
+    Example:
+        >>> from clarifai.client.model import Model
+        >>> model = Model("url")
+        >>> model.export('/path/to/export_model_dir')
+    """
+    assert self.model_info.model_version.id, "Model version is empty. Please provide `model_version` as arguments or with a URL as the format '{user_id}/{app_id}/models/{your_model_id}/model_version_id/{your_version_model_id}' when initializing."
```
Comment (wemoveon2) [inline clarifai/client/model.py:773]:
```suggestion
    assert self.model_info.model_version.id, "Model version ID is missing. Please provide a `model_version` with a valid `id` as an argument or as a URL in the following format: '{user_id}/{app_id}/models/{your_model_id}/model_version_id/{your_version_model_id}' when initializing."
```

**Review by wemoveon2 — COMMENTED** [review]
has this been manually tested? unit tests?

**Comment by sainivedh** [comment]
> has this been manually tested? unit tests?

I've tested it manually, for SDK we don't have slow tests as it takes around 10-15 min to train and export a model.

**Pull request merged**